Assets/MRTK/Core/Inspectors/Utilities/MixedRealityInspectorUtility.cs

```diff
@@ -562,7 +583,8 @@ public static bool DrawProfileDropDownList(SerializedProperty property, BaseMixe
                     profileContent,
                     GUILayout.ExpandWidth(true));
 
-                property.objectReferenceValue = (newIndex > 0) ? profileInstances[newIndex - 1] : null;
+                int profileInstanceIndex = newIndex - dropdownOffset;
+                property.objectReferenceValue = (profileInstanceIndex >= 0) ? profileInstances[profileInstanceIndex] : null;
```

Comment:
How does this behave if no profile assets are found? Does it just say `null` or something in the dropdown?

Reply:
It breaks currently (yikes!) Making a fix for this case now

Reply:
Addressed this case and added a warning message, let me know if it should be worded to be clearer!

![image](https://user-images.githubusercontent.com/39840334/111009316-22395a00-8348-11eb-862d-6aa85c87b8ae.png)